xdsl/dialects/x86/ops.py

```diff
@@ -688,11 +741,11 @@ class RM_XorOp(R_RM_Operation[GeneralRegisterType, GeneralRegisterType]):
 
 
 @irdl_op_definition
-class RM_MovOp(R_RM_Operation[GeneralRegisterType, GeneralRegisterType]):
+class RM_MovOp(R_M_Operation[GeneralRegisterType, GeneralRegisterType]):
     """
-    Copies the value from the memory location pointed to by r2 into r1.
+    Copies the value from the memory location pointed to by operand register into output register.
```

Comment:
we've been using "source" and "destination" in riscv and asm, would that make sense as a description in x86 land also?